build.gradle

```diff
@@ -65,36 +65,60 @@ subprojects {
     version = getProductVersion()
 
     ext {
-        dropwizardVersion = '1.3.14'
+        awaitilityVersion = '3.1.6'
+        commonsCliVersion = '1.4'
+        commonsCodecVersion = '1.11'
+        commonsIoVersion = '2.6'
+        commonsMathVersion = '3.6.1'
+        databaseRiderVersion = '1.5.2'
+        dropWizardTestingVersion = '1.3.12'
+        dropwizardVersion = '1.3.15'
```

Comment:
Note, dropwizard version updated here from 1.3.14 to 1.3.15

For the most part this is just a refactoring update, there are one or two other places where version numbers were brought inline.